testing/README.md

```diff
@@ -278,6 +278,306 @@ Clears all registered benchmarks, so calling `runBenchmarks()` after it wont run
+tests. The `it` function registers an individual test case. The `describe` and
+`it` functions have similar call signatures to `Deno.test`, making it easy to
+migrate from using `Deno.test`.
+
+### Hooks
+
+There are 4 types of hooks available for test suites. A test suite can have
+multiples of each type of hook, they will be called in the order that they are
+registered. The `afterEach` and `afterAll` hooks will be called whether or not
+the test case passes. The all hooks will be called once for the whole group
+while the each hooks will be called for each individual test case.
+
+- `beforeAll`: Runs before all of the tests in the test suite.
+- `afterAll`: Runs after all of the tests in the test suite finish.
+- `beforeEach`: Runs before each of the individual test cases in the test suite.
+- `afterEach`: Runs after each of the individual test cases in the test suite.
+
+If a hook is registered at the top level, a global test suite will be registered
+and all tests will belong to it. Hooks registered at the top level must be
+registered before any individual test cases or test suites.
+
+### Focusing tests
+
+If you would like to only run specific individual test cases, you can do so by
+calling `it.only` instead of `it`. If you would like to only run specific test
+suites, you can do so by calling `describe.only` instead of `describe`.
+
+There is one limitation to this when the individual test cases or test suites
+belong to another test suite, they will be the only ones to run within the top
+level test suite.
+
+### Ignoring tests
+
+If you would like to not run specific individual test cases, you can do so by
+calling `it.ignore` instead of `it`. If you would like to only run specific test
+suites, you can do so by calling `describe.ignore` instead of `describe`.
+
+### Sanitization options
+
+Like `Deno.TestDefinition`, the `DescribeDefinition` and `ItDefinition` have
+sanitization options. They work in the same way.
+
+- sanitizeExit: Ensure the test case does not prematurely cause the process to
+  exit, for example via a call to Deno.exit. Defaults to true.
+- sanitizeOps: Check that the number of async completed ops after the test is
+  the same as number of dispatched ops. Defaults to true.
+- sanitizeResources: Ensure the test case does not "leak" resources - ie. the
+  resource table after the test has exactly the same contents as before the
+  test. Defaults to true.
+
+### Permissions option
+
+Like `Deno.TestDefinition`, the `DescribeDefintion` and `ItDefinition` have a
+permissions option. They specify the permissions that should be used to run an
+individual test case or test suite. Set this to "inherit" to keep the calling
+thread's permissions. Set this to "none" to revoke all permissions.
+
+Defaults to "inherit".
+
+There is currently one limitation to this, you cannot use the permissions option
+on an individual test case or test suite that belongs to another test suite.
+
+### Migrating and usage
```

Comment:
Comparing `describe / it` APIs with Deno.test is very good, but we generally don't recommend migrating from `Deno.test` suites to BDD style testing (`Deno.test` should still be the default way of writing tests and BDD style should be an optional convenient way of writing them if the users prefer) So the section name here sounds concerning to me.

Could you rephrase the section name to something like `Comparing to Deno.test`?